sysutils/dokku/Portfile

```
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+
+github.setup            dokku dokku 0.29.0 v
+github.tarball_from     archive
+
+categories              sysutils
+
+maintainers             {@bashu gmail.com:bashu.was.here} openmaintainer
+
+description             Command-line client for the Dokku PaaS
+
+long_description        ${description}
```

Reusing the `${description}` variable like this requires the use of Tcl's expansion operator (`{*}`), so this should be written like so:

```suggestion
long_description        {*}${description}
```

---

fixed